operators-manual/companion-web.md

```diff
@@ -198,6 +198,20 @@ Apart from the four pages that can be accessed via direct links, Companion web i
 
 <img src="/images/git.png" class="img-responsive img-center" style="max-width:750px" />
 
+#### Manual updates
+
+> <span style="color:red">This can result in an unstable behaviour.</span>
+
+Steps to manual update:
+1. Fetch the last changes of the remote repository.
```

Comment:
1. make sure you have internet access at https://192.168.2.2:2770/network
1. go to http://192.168.2.2:2770/git
1. click the large black text of the remote you want ex `origin`
1. click 'tags' or 'branches' to display available refs
1. click the ref you want to update to, it will be highlighted yellow when selected
1. click 'update' and wait for the update to complete by monitoring the output in the text area